app/assets/javascripts/audit_logs.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Your API app probably doesn't have CSS and JavaScript so you can remove all of these files. Also because they are empty. :)